tools/topology/sof-cml-rt5682-max98357a.m4

```diff
@@ -0,0 +1,76 @@
+#
+# Topology for Cometlake with rt5682 headset on SSP0, max98357a spk on SSP1
+#
+# Adding max98357a spk on SSP1 on top of sof-cml-rt5682.
+#
+
+# Include topology builder
+include(`utils.m4')
+include(`dai.m4')
+include(`pipeline.m4')
+include(`ssp.m4')
+
+# Include TLV library
+include(`common/tlv.m4')
+
+# Include Token library
+include(`sof/tokens.m4')
+
+# Include Cannonlake DSP configuration
+include(`platform/intel/cnl.m4')
+include(`platform/intel/dmic.m4')
```

Comment:
Do we need all the above includes ? (since they will already be included by sof-cml-rt5682.m4).

Reply:
makes sense.. will remove.